File: CSharp/1553.cs
using System; 
using System.Linq;
using System.Collections.Generic;

class URI {

    static void Main(string[] args) 
    { 
        Dictionary<int,int> dic = new Dictionary<int,int>();
        int[] listao;
        int[] n;
        int cont=0;

        while(true)
        {
            n = Console.ReadLine().Split().Select(int.Parse).ToArray();
            if(n[0]==0 && n[1]==0)
            {break;}
            listao = Console.ReadLine().Split().Select(int.Parse).ToArray();

            foreach(int i in listao)
            {
                if(dic.ContainsKey(i))
                {
                    dic[i]++;
                }
                else
                {
                    dic[i]=1;
                }
            }
            foreach(var x in dic.Values)
            {
                if(x>=n[1])
                {cont++;}
            }
            Console.WriteLine(cont);
            cont=0;
            dic.Clear();
        }
    }
}

File: Python/1553.py
dic = {}
cont = 0

while True:
    n = list(map(int, input().split()))
    if n[0] == 0 and n[1] == 0:
        break
    listao = list(map(int, input().split()))

    for i in listao:
        if i in dic:
            dic[i] += 1
        else:
            dic[i] = 1

    for x in dic.values():
        if x >= n[1]:
            cont += 1

    print(cont)
    cont = 0
    dic.clear()
